### ISSUE TITLE
xfwm-compact and xfwm-a11y not correctly installed

### ISSUE THREAD

**Issue by m4u9** [shimmerproject/Greybird#307]
I noticed that the xfwm-compact and xfwm-a11y no longer show up in the window manager theme settings. I think it's because they are installed in the wrong path. Can anybody confirm?

**Comment by paulstodghill** [comment]
I have the same problem on Xubuntu 21.10. It looks like the problem exists in Debian Sid as well. Both versions are 3.22.15-1.
Here's my work around:

```
mkdir -p ~/.themes/Greybird-accessibility
mkdir -p ~/.themes/Greybird-compact
mkdir -p ~/.themes/Greybird-dark-accessibility
ln -s /usr/share/themes/Greybird-accessibility/xfwm4-a11y ~/.themes/Greybird-accessibility/xfwm4
ln -s /usr/share/themes/Greybird-compact/xfwm4-compact ~/.themes/Greybird-compact/xfwm4
ln -s /usr/share/themes/Greybird-dark-accessibility/xfwm4-a11y ~/.themes/Greybird-dark-accessibility/xfwm4
```

**Comment by m4u9** [comment]
I think I identified the issue in the meson file  at https://github.com/shimmerproject/Greybird/blob/master/light/meson.build#L70

Currently:
`install_subdir('xfwm4-a11y', install_dir : a11y_theme_path)` 

should instead be:

`install_data(xfwm4_a11y_files, install_dir : join_paths (a11y_theme_path, 'xfwm4'))`

I will run some tests and then push a commit

**Comment by tux-ware** [comment]
The issue is still present in Xubuntu Core 22.04 with the latest updates. Below, you can see that paths of the first 3 "Greybird-" themes are not ending with xfwm4 dir - these themes are not showing in the window manager theme settings:

dave@dell-dawida:/usr/share/themes$ ls -ld */xfwm*/
drwxr-xr-x 1 root root 4830 feb 19 09:41 Daloa/xfwm4/
drwxr-xr-x 1 root root 4830 feb 19 09:41 Default-hdpi/xfwm4/
drwxr-xr-x 1 root root 4830 feb 19 09:41 Default/xfwm4/
drwxr-xr-x 1 root root 4830 feb 19 09:41 Default-xhdpi/xfwm4/
drwxr-xr-x 1 root root 4250 feb 19 09:41 Greybird-accessibility/xfwm4-a11y/
drwxr-xr-x 1 root root 4088 feb 19 09:41 Greybird-compact/xfwm4-compact/
drwxr-xr-x 1 root root 2416 feb 19 09:41 Greybird-dark-accessibility/xfwm4-a11y/
drwxr-xr-x 1 root root 2190 feb 19 09:41 Greybird-dark/xfwm4/
drwxr-xr-x 1 root root 4024 feb 19 09:41 Greybird/xfwm4/
drwxr-xr-x 1 root root 4830 feb 19 09:41 Kokodi/xfwm4/
drwxr-xr-x 1 root root 4070 feb 19 09:41 Moheli/xfwm4/
drwxr-xr-x 1 root root 2746 feb 19 09:41 Numix/xfwm4/
drwxr-xr-x 1 root root 1476 mar 18 12:26 Shiki-Colors-Easy-Metacity/xfwm4/
drwxr-xr-x 1 root root 1476 mar 18 12:26 Shiki-Colors-Metacity/xfwm4/

The workaround provided by @paulstodghill solves it.

**Comment by bluesabre** [comment]
This should be resolved by https://github.com/shimmerproject/Greybird/pull/318.

**Comment by bluesabre** [comment]
This has now been resolved with the merge of #318
